packages/as-types/assembly/result.ts

```diff
@@ -6,32 +6,42 @@
  */
```

Comment:
Can you improve the top documentation by taking inspiration from massa-as-sdk Address class

Reply:
yes